google/cloud/opentelemetry/monitoring_exporter.cc

```diff
@@ -57,17 +67,14 @@ class MonitoringExporter final
  private:
   opentelemetry::sdk::common::ExportResult ExportImpl(
       opentelemetry::sdk::metrics::ResourceMetrics const& data) {
-    auto request = ToRequest(data);
+    auto request = ToRequest(data, prefix_);
     request.set_name(project_.FullName());
     if (request.time_series().empty()) {
       GCP_LOG(WARNING) << "Cloud Monitoring Export skipped. No TimeSeries "
                           "added to request.";
       return opentelemetry::sdk::common::ExportResult::kSuccess;
     }
 
-    // TODO(#13869) - Exporters of Google-defined metrics will need to use
```

Comment:
nit: unrelated?

Reply:
Correct, I got my commits twisted. Restored and updated the linked issue.